src/lib/Behat/BusinessContext/ContentUpdateContext.php

```diff
@@ -28,6 +30,15 @@ public function iSetFields(TableNode $table): void
         }
     }
 
+    /**
+     * @When I select :contentPath from Image Asset Repository for :fieldName field
+     */
+    public function selectContentFromIARepository(string $contentPath, string $fieldname): void
```

Comment:
Nitpick: `fieldname` is not a single word, so we should write it in camel case.
Examples from kernel: https://github.com/ezsystems/ezpublish-kernel/search?q=%24fieldName&unscoped_q=%24fieldName